.studiorc

```diff
@@ -127,3 +176,104 @@ function install_cookstyle() {
   RUBY_BIN="$(hab pkg path core/ruby)/bin"
   export PATH="$RUBY_BIN:$PATH"
 }
+# Adding auto tab complete
+complete -F _install_cookstyle
+
+document "pre_commit_verify" <<DOC
+  Runs all verify steps on local code before commiting or creating pr.
+DOC
+function pre_commit_verify() {
```

Comment:
I don't see this function used anywhere. Where is the plan to run it?

Reply:
In studio prior to submitting a pr.